libdnf-cli/utils/utf8.cpp

```diff
@@ -159,11 +163,18 @@ std::string substr_width(const std::string & str, std::string::size_type pos, st
 
         // increase string width
         if (wid != std::string::npos) {
-            std::size_t char_width = wcwidth(wide_char);
-            if (char_width > wid) {
-                break;
+            auto char_width = wcwidth(wide_char);
+            if(char_width >= 0){
+                auto new_wid = wid - static_cast<std::make_unsigned<decltype(char_width)>::type>(char_width);
+                if (new_wid > 0) {
+                    break;
+                }
+                else{
+                    wid = static_cast<std::make_unsigned<decltype(new_wid)>::type>(new_wid);
+                }
```

Comment:
`make test` errors out on test_libdnf_cli; I believe there's an issue in this code.